### DISCUSSION TITLE
Can we connect to titanium proxy second proxy?

### DISCUSSION THREAD

**Discussion by vipvodu** [justcoding121/Titanium-Web-Proxy#833] [General]
1. Have titanium proxy 127.0.0.1:9032 
2, Have other proxy with login and password

Browser connect to 127.0.0.1:9032 , but need that connect with domen will be from ip second proxy with login and pass

Its posible?

**Comment by vipvodu** [comment]
now find info about ExternalProxy. work fine, Thanks!

**Comment by justcoding121** [comment]
Yep ? `ExternalProxy` is exactly for that. Glad you got it working.

```csharp
var upstream = new ExternalProxy("second-proxy-host", 8080)
{
    UserName = "user",
    Password = "pass"
};
proxyServer.UpStreamHttpProxy = upstream;
proxyServer.UpStreamHttpsProxy = upstream;
```

Browser ? Titanium `:9032` ? upstream proxy ? site. Closing.
